src/utils/map.ts

```diff
@@ -225,9 +225,18 @@ const getWMSLayerUrlsAndBounds = async (
         },
       } = parsedResponse;
 
-      const filteredLayers = Layer.filter((L: WmsLayer) =>
-        allLayerNames.includes(L.Name)
-      );
+      let filteredLayers = [];
+      if (Object.prototype.toString.call(Layer) === '[object Array]') {
```

Comment:
Does this check if `Layer` is an array? It could be done like this:
```suggestion
      if (Array.isArray(Layer)) {
```